apps/emqx/src/emqx_router_syncer.erl

```diff
@@ -149,10 +189,34 @@ stats() ->
 
 %%
 
-init([Pool, Id]) ->
-    true = gproc_pool:connect_worker(Pool, {Pool, Id}),
-    {ok, #{stash => stash_new()}}.
+mk_state(Options) ->
+    #{
+        state => maps:get(initial_state, Options, open),
```

Comment:
maybe `open` -> `active`, and `closed` -> `prohibited` or

Reply:
Renamed to active/suspended.